code/game/gamemodes/changeling/modularchangling.dm

```diff
@@ -190,6 +190,13 @@ var/list/datum/power/changeling/powerinstances = list()
 	genomecost = 8
 	verbpath = /mob/proc/changeling_rapidregen
 
+/datum/power/changeling/armblade
+	name = "Arm Blade"
+	desc = "We transform one of our arms into an organic blade that can cut through flesh and bone."
+	helptext = "The blade can be retracted by using the same verb used to manifest it. It has a chance to deflect projectiles."
```

Comment:
I don't see anything in the armblade's functionality that would make it deflect projectiles, to do that you would need to add the following where the armblade is defined:
```
/obj/item/weapon/armblade/IsShield()
return 1
```